StripePaymentSheet/StripePaymentSheet/Source/Categories/STPPaymentMethod+PaymentSheet.swift

```diff
@@ -23,7 +23,11 @@ extension STPPaymentMethod {
             // The missing space is not an oversight, but on purpose
             return "••••\(usBankAccount?.last4 ?? "")"
         default:
-            return type.displayName
+            if let linkPaymentDetails {
+                return "•••• \(linkPaymentDetails.last4)"
```

Comment:
Is it guaranteed that all link payment method will always have a last 4? Do we want to format them differently for Link payment methods that are banks or cards etc?

Reply:
Yep, cards (which `linkPaymentDetails` currently represents) will always have a last 4.

Are you ok with keeping this as-is and turning `LinkPaymentDetails` into an enum later on as we add support for bank accounts?

We could then also tweak the rendering to be `•••• 4242` for cards and `••••6789` for bank accounts.